alma/lib/ShareOfCheckout/ShareOfCheckoutHelper.php

```diff
@@ -74,7 +74,7 @@ public function __construct(
     public function shareDays()
     {
         $shareOfCheckoutEnabledDate = $this->getEnabledDate();
-        if (!Settings::canShareOfCheckout() || empty($shareOfCheckoutEnabledDate)) {
+        if (Settings::canShareOfCheckout() == ShareOfCheckoutAdminFormBuilder::ALMA_SHARE_OF_CHECKOUT_CONSENT_NO || empty($shareOfCheckoutEnabledDate)) {
```

Comment:
when there is multiple condition i think it's more readable to put them on different lines like
 ```php
if(
  Settings::canShareOfCheckout()==ShareOfCheckoutAdminFormBuilder::ALMA_SHARE_OF_CHECKOUT_CONSENT_NO
   || empty($shareOfCheckoutEnabledDate)
) {`